docs/website/docs/deployment-configurations/cpu-dylib.md

```diff
@@ -93,6 +93,7 @@ In the build directory, run the following command:
 
 ``` shell hl_lines="3"
 iree/tools/iree-translate \
+    -iree-input-type=mhlo \
```

Comment:
This one shouldn't be needed. The TensorFlow importer no longer stops at `mhlo`.

Reply:
Fixed.